src/core/a-animation.js

```diff
@@ -356,13 +357,14 @@ function getAnimationValues (el, attribute, dataFrom, dataTo, currentValue) {
   var from = {};
   var partialSetAttribute;
   var to = {};
-
-  if (attributeSplit.length === 2) {
+  if (attributeSplit.length === 2 && attributeSplit.indexOf('color') === -1) {
```

Comment:
If the attribute can be split, then you can use `el.components[attrSplit[0]].schema[attrSplit[1]].type === 'color'`.

Else you can let it fall thru the if/else statements with the `isNaN`.